src/app/main/main.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="mainPage">
+    <div class="background">
+        <div class="values">
+            <h1>Educate</h1>
+            <h1>Innovate</h1>
+            <h1>Inspire</h1>
+        </div>
+        <div class="b">
+            <a href="../cars" class="button">The Schulich Elysia <span>&#x203A;</span></a>
```

Comment:
If you update this branch to the current version of master, you can make this link go to the elysia page, rather that the cars page. (so ../elysia).